projects/assets/src/translations/en/product.ts

```diff
@@ -36,6 +36,7 @@ export const product = {
     id: 'ID',
     showReviews: 'Show reviews',
     share: 'Share',
+    outOfStock: 'Out of stock',
```

Comment:
Not a big deal, but is there a reason why the solution differs from the suggested fix ini the ticket description?
> In cart-item.component.html replace productSummary.outOfStock with addToCart.outOfStock.

Reply:
I'm not involved but I got an email as the guy who reported the issue 😂 I have no preference.

Reply:
`addToCart.outOfStock` translation key already exists in `cart` translation block, it would be cleaner to update the template to use the translation key for the `cart` rather then creating the new one in `product` which creates unnecessary coupling

Reply:
I quickly read over the ticket. You're right I'll make the change lol.